src/components/output/custom-els/MultiPanel/styles.css

```diff
@@ -0,0 +1,11 @@
+multi-panel > .panel-heading {
+  background:gray;
+}
+multi-panel > .panel-content {
+  height:0px;
+  overflow:scroll;
+  transition: height 1s;
+}
+multi-panel > .panel-content[expanded] {
+  height:30px;
```

Comment:
Why 30?

Reply:
no real reason for 30px. 
This will be the size of the expanded panel which is application design dependent. I was thinking user can override this ? it just need some kind of value for height and this is "placeholder default"